elastic/apm-server
# [modelindexer] collect additional metrics

## Motivation/summary

collect metrics on events processed partitioned by
service name and service version.

## Checklist


For functional changes, consider:
- Is it observable through the addition of either **logging** or **metrics**?
- Is its use being published in **telemetry** to enable product improvement?
- Have system tests been added to avoid regression?

## How to test these changes

1. start apm-server
2. index events
3. confirm that metrics exist showing the number of events processed by service.name and service.version
